tools/code-analyzer/src/commands/analyzer/index.ts

```diff
@@ -47,6 +47,11 @@ export default class Analyzer extends Command {
 				'GitHub branch or commit hash to compare against the base branch/commit.',
 			required: true,
 		},
+		{
+			name: 'sinceVersion',
+			description:
+				'Specify the version used to determine which changes are included (version listed in @since code doc).',
+		},
```

Comment:
The `required` [prop](https://oclif.io/docs/args) defaults to `false`, so omitting this argument gives a cryptic error.

Reply:
Whoops! forgot to mark required. Fixed!